client/state/posts/utils.js

```diff
@@ -31,7 +31,9 @@ import stripHtml from 'lib/post-normalizer/rule-strip-html';
  * Constants
  */
 
-const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/;
+const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/,
+	REGEXP_EMPTY_CONTENT = /^<p>(<br[^>]*>|&nbsp;|\s)*<\/p>$/,
```

Comment:
Is there a specific reason that `&nbsp;` is chosen here? Do we automatically insert that characters? There are many other space characters that I could imagine being in the same group here, even hex-coded or decimal-coded equivalent entities.